prototypes/updates/base/recipes.lua

```diff
@@ -26,6 +26,11 @@ data_util.add_or_replace_ingredient(
   "iron-plate",
   { type = "item", name = "iron-beam", amount = 4 }
 )
+data_util.add_or_replace_ingredient(
+  "assembling-machine-1",
+  "electronic-circuit",
```

Comment:
I saw now that there is some code in `prototypes/updates/add-automation-core-recipes.lua` but it seems like it does not trigger and that is why it broke, I suppose I could try to figure out what is wrong